addons/mail/static/src/component_hooks/use_should_update_based_on_props/use_should_update_based_on_props.js

```diff
@@ -5,41 +5,21 @@ const { Component } = owl;
 const { onPatched } = owl.hooks;
 
 /**
- * Compares `a` and `b` up to the given `propsCompareDepth`.
+ * Shallow compares `a` and `b`.
  *
  * @param {any} a
  * @param {any} b
```

Comment:
a and b are now always props, so this doc could become Object instead (even before this PR I think it was really only Object or Array and never anything else)